src/bumprace.c

```diff
@@ -1218,7 +1218,10 @@ if (client && (!(skt = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP)) )) {
   sprintf(text,"%s/font.scl",DATAPATH);
   Font=LoadImage("font.png",3);
   InitFont(Font);
-  if (final && client != 2) StartText(i);
+#ifdef NET
+  if (client != 2)
+#endif //NET
```

Comment:
You could set `client = 0` once if compiled without NET, then you would not need the ifdef here (and below) and could keep the single if.

Reply:
I'Il just leave it like this, it is ugly, but it does keep all the networking code together.

Reply:
getting the server hostaddresss is quite difficult since it can have multiple interfaces and ip addresses.